ndb/src/google/cloud/ndb/async_.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2018 Google LLC
```

Comment:
Do you want `async_` to be public? Could we start with it non-public and then "upgrade" if needed?

Reply:
Well, the async_context needs to be public. Otherwise I would have just put it in _eventloop.

Reply:
We can make the desired members public by importing them into `__init__.py`?

Reply:
Done.